dev_guide/deployments.adoc

```diff
@@ -109,6 +109,30 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 <5> The link:#rolling-strategy[rolling strategy] is the default and may be omitted.
 ====
 
+== Deployment Client Tool Operations
+Use the 'ocs deploy' command with available options to start, view, cancel, or
+retry a deployment.
+
+----
+$ oc deploy <deployment_config> [--options]
+----
+
+[cols= 2]
+|===
+|`oc deploy <deployment_config> --latest`
+|Starts the specified deployment
+
+|`oc deploy <deployment_config>`
+|Dsiplays the specified deployment
+
+|`oc deploy <deployment_config> --retry`
+|Attempts to retry to deploy the last deployment. The deployer pod and any hook pods are deleted.
+
+|`oc deploy <deployment_config> --cancel`
+|Cancels the deployment that is currently in progress
```

Comment:
Agree, all good ideas. Thanks! I say correct the minor typos I mentioned (and any of the other small changes you agree with, I don't feel very strongly at this point) and we'll iterate from there.